core/modules/layout/layout.module

```diff
@@ -564,13 +572,22 @@ function layout_permission() {
       'description' => t('Add and modify layouts on pages.'),
       'restrict access' => TRUE,
     ),
+    'use all templates' => array(
+      'title' => t('Use disabled templates'),
```

Comment:
Include the word "layout" here (`t('Use disabled layout templates')`) and in the description below in order to not be vague (in the future we might need to refer to other types of templates in the UI). Will also help when we add a search filter in the permissions page (https://github.com/backdrop/backdrop-issues/issues/980)